modules/osdk-ensuring-operator-workloads-run-restricted-psa.adoc

```diff
@@ -15,16 +15,45 @@ If you ensure your Operator workloads can run in namespaces set to the `restrict
 
 .Procedure
 
-* To configure Operator workloads to run in namespaces set to the `restricted` pod security level, edit the namespace definition in your as shown in the following example:
+* To configure Operator workloads to run in namespaces set to the `restricted` pod security level, edit your Operator's namespace definition similar to the following examples:
++
+[IMPORTANT]
+====
+It is recommended that you set the seccomp profile in your Operator's namespace definition. However, setting the seccomp profile is not supported in {product-title} 4.10 and earlier.
+====
+
+** For Operator projects that must run in the currently supported  {product-title} versions, including 4.10 and earlier, edit your Operator's namespace definition similar to the following example:
++
+.Example `config/manager/manager.yaml` file
+[source,yaml]
+----
+...
+spec:
+ securityContext:
+   seccompProfile:
+     type: '' <1>
```

Comment:
i don't think this is valid, was it tested?  i think you need to leave out the seccompProfile field entirely in order to leave it "empty"

Reply:
Addressed in 6946ad6. To make it easier to understand, I swapped the steps for >=4.11 and <=4.10. It was a little odd annotating the absence of a field. Please let me know if this seems legible to you.